File: app/src/main/java/io/plan8/backoffice/vm/BaseVM.java
package io.plan8.backoffice.vm;

import android.content.Context;
import android.databinding.BaseObservable;
import android.os.Bundle;

/**
 * Created by chokwanghwan on 2017. 11. 28..
 */

public class BaseVM extends BaseObservable {
    private Context context;
    private Bundle savedInstanceState = null;

    public BaseVM(Context context, Bundle savedInstanceState) {
        this.context = context;
        this.savedInstanceState = savedInstanceState;
    }

    public BaseVM(Context context) {
        this.context = context;
    }
}

File: app/src/main/java/io/plan8/business/vm/BaseVM.kt
package io.plan8.business.vm

import android.content.Context
import android.databinding.BaseObservable
import android.os.Bundle

/**
 * Created by SSozi on 2017. 11. 2..
 */
open class BaseVM : BaseObservable {
    protected var context: Context
    var savedInstanceState: Bundle? = null

    constructor(context: Context, savedInstanceState: Bundle?) {
        this.context = context
        this.savedInstanceState = savedInstanceState
    }

    constructor(context: Context) {
        this.context = context
    }
}